src/pages/leejiyon/Components/Nav.scss

```
@@ -0,0 +1,107 @@
+@import '/src/styles/Variables.scss';
+
+.NavJY {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 54px;
+  width: 100%;
+  padding: 0 83px;
+  background-color: #fff;
+  border-bottom: 1px solid $border-color;
+  z-index: 100;
+
+  .navLogo {
+    position: relative;
+    display: flex;
+
+    img {
+      width: 20px;
+      height: 20px;
+      margin-right: 20px;
+    }
+
+    .navLogoTitle {
+      line-height: 20px;
+      font-size: 20px;
+      font-family: 'Lobster', cursive;
+    }
+  }
+
+  .navLogo .navLogoTitle::before {
+    content: '';
+    position: absolute;
+    left: 30px;
+    top: 50%;
+    transform: translateY(-50%);
+    display: block;
+    width: 1px;
+    height: 20px;
+    background-color: black;
+  }
+
+  .navSearchWrap {
+    position: relative;
+  }
+
+  .navSearch {
+    width: 215px;
+    height: 28px;
+    background-color: #fafafa;
+    border: 1px solid #dbdbdb;
+    border-radius: 3px;
+    text-align: center;
+  }
+
+  @media screen and (max-width: 650px) {
+    .navSearch {
+      display: none;
+    }
+  }
+
+  .searchPopup {
+    position: absolute;
+    top: 35px;
+    left: 50%;
+    transform: translateX(-50%);
+    width: 375px;
+    background-color: #fff;
+    border: 1px solid var(--border-color);
+    border-radius: 5px;
+    overflow: auto;
+    z-index: 100;
+  }
+
+  .searchPopup li {
+    display: block;
+    display: flex;
+    margin: 10px;
+  }
+
+  .searchPopup li img {
+    width: 30px;
+    height: 30px;
+    margin-right: 10px;
+    border-radius: 50%;
+  }
+
+  .navLink {
+    min-width: 130px;
+
+    li {
+      cursor: pointer;
+    }
+  }
+
+  .navLink img {
+    width: 22px;
+    margin-left: 10px;
+  }
+
+  li {
+    display: inline-block;
+  }
+}
```

js 파일에 구성으로 봐서는 ul 태그(className="navLink") 안에 li 태그 안에 img 태그가 위치하고 있어서 아래와 같이 하시면 좋을것 같아요.

```suggestion
 .navLink {
    li {
        display: inline-block;       
        img {
          width: 22px;
          margin-left: 10px;
        }
     }    
  }
```

---

꼼꼼한 지적 감사합니다